### ISSUE TITLE
Connl 2003 NER task Flair F1 score

### ISSUE THREAD

**Issue by alipetiwala** [flairNLP/flair#754]
For the Connl 2003 NER task Flair has reported an F1-score 93.09 ±0.12% for test set , did you use both training and dev data as training set? Do you have test scores only if you use training set for training ?

**Comment by alanakbik** [comment]
Hello @alipetiwala we do parameter selection against the dev data (i.e. train on train data, evaluate on dev). See also the included [hyperopt](https://github.com/zalandoresearch/flair/blob/master/resources/docs/TUTORIAL_8_MODEL_OPTIMIZATION.md) package to automate this. Then with the best parameter combination we do 5 seperate runs in which we train using both train and dev data and evaluate on test data. We average reported numbers of all 5 runs and report standard deviation. 

The current best reported numbers for CoNLL-03 can be found [here](https://github.com/zalandoresearch/flair/blob/master/resources/docs/EXPERIMENTS.md), along with instructions on how to reproduce them. I don't have numbers for using training set only at hand, but I think they were something around 92.6 for regular `FlairEmbeddings`. We never did train-only experiments for `PooledFlairEmbeddings`, but you can get these numbers by running Flair with the appropriate configuration.
